local-storage/operator/base/operator-group.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: local-storage-operator-group
+  namespace: openshift-local-storage
```

Comment:
Can you pull all of the namespaces from the objects and set it in the kustomize file instead?

Reply:
Updated